crates/hdk/Cargo.toml

```diff
@@ -30,7 +30,7 @@ serde_bytes = "0.11"
 thiserror = "1.0.22"
 tracing = "0.1"
 tracing-core = "0.1"
-mockall = { version = "0.9", optional = true }
+mockall = { version = "0.8", optional = true }
```

Comment:
why downgrading?

Reply:
Just to be uniform with the others. I tried upgrading to 0.10 but it broke everything so I just set them all back to 0.8